.gitlab-ci.yml

```diff
@@ -67,10 +71,12 @@ generate_integration_tests:
   stage: generate_integration_test
   script:
     # extract package name
-    - export CI_DEPENDENCY_NAME=$(cat $CI_PROJECT_DIR/Project.toml | grep name | awk '{ print $3 }' | tr -d '"')
-    - echo "CI_DEPENDENCY_NAME -> $CI_DEPENDENCY_NAME"
     - apt update && apt install -y git
-    - git clone --depth 1 -b dev https://github.com/QEDjl-project/QuantumElectrodynamics.jl.git /QEDjl
+    - git clone --depth 1 -b setDevDevCompat https://github.com/SimeonEhrig/QED.jl.git /QEDjl
```

Comment:
```suggestion
    - git clone --depth 1 -b setDevDevCompat https://github.com/QEDjl-project/QED.jl.git /QEDjl
```

Reply:
You're right, and also the repository is now QuantumElectrodynamics.jl not QED.jl ^^

Reply:
Should I change it again?

Reply:
I fixed it. But at some point, it should become obsolete because I want to write the CI bootloader.